.github/workflows/docker.yml

```diff
@@ -44,7 +51,8 @@ jobs:
           platforms: linux/amd64,linux/arm64,linux/386
```

Comment:
We'd maybe want to specify this as a job level environment variable if possible, but think we can conditionally specify a platform here with something like

```suggestion
          platforms: ${{ env.DOCKER_PUSH && 'linux/amd64,linux/arm64,linux/386' || 'linux/amd64' }}
```